src/sentry/tasks/reports.py

```diff
@@ -34,13 +43,50 @@
 from sentry.utils.http import absolute_uri
 from sentry.utils.iterators import chunked
 from sentry.utils.math import mean
+from sentry.utils.outcomes import Outcome
+from sentry.utils.snuba import raw_snql_query
 
 date_format = partial(dateformat.format, format_string="F jS, Y")
 
 logger = logging.getLogger(__name__)
 
 BATCH_SIZE = 20000
 
+ONE_DAY = 60 * 60 * 24
```

Comment:
Made this a constant for readability.

Reply:
Maybe we already have something like this?

Reply:
It's kind of built-in: `int(timedelta(days=1).total_seconds)`

Reply:
sentry.api.base has this too.

Reply:
yeah dis good pattern. I miss Go's nice `Duration`s

```go
const (
    Nanosecond  Duration = 1
    Microsecond          = 1000 * Nanosecond
    Millisecond          = 1000 * Microsecond
    Second               = 1000 * Millisecond
    Minute               = 60 * Second
    Hour                 = 60 * Minute
)
```